src/Services/AuthWrapper.js

```diff
@@ -1,34 +1,29 @@
-/* eslint-disable react/jsx-props-no-spreading */
-import React, { useEffect } from 'react';
-import { useNavigate, useParams, useLocation } from 'react-router-dom';
+import { useEffect } from 'react';
+import { useNavigate, useLocation } from 'react-router-dom';
 
-const isAuthenticated = () => {
+export const isAuthenticated = () => {
```

Comment:
@tackc do we need to export this? I think the AuthWrapper should be the only thing we need to export

Reply:
It's used on `ErrorPage` to conditionally render the <Navbar /> component. I tried removing it and the app didn't compile